Benchmarks/Fibonacci/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "emitEntryPoint": true,
+    "compilerName": "fsc",
+    "define": [ "TRACE" ],
+    "xmlDoc": true,
+    "compile": {
+      "includeFiles": [
+        "Fibonacci.fs"
+      ]
+    },
+    "additionalArguments": [
+      "--warnon:1182"
+    ]
+  },
+  "runtimeOptions": {
+    "configProperties": {
+      "System.GC.Server": true
+    }
+  },
+  "dependencies": {
+    "FSharp.Core": "4.0.1.7-alpha",
+    "Hopac": { "target": "project" },
+    "Hopac.Platform": { "target": "project" }
+  },
+  "frameworks": {
+    "net46": {},
```

Comment:
yes the msbuild for net46 should be the truth.
i added `net46` in project.json for two reason:

- msbuild use latest rtm compiler, in project.json instead use latest dev compiler. We got a bug before fixed by @ncave but i was trying to diagnose if was a compiler issue (happen sometimes).
- having both `net46` and `netstandard1.6` i can run multijit/fw (ryujit, legacyjit, coreclr, fullcorl)  tests like in `FibonacciMultijit`, because BenchamrkDotNet having all info needed in project.json (refs), can build/run same benchmark test easy (see the table in my previous comment https://github.com/Hopac/Hopac/pull/118#issuecomment-244135453 )

if you remove `net46` you lose both.
at dev time there is not difference, only `dotnet build` will build both fw, instead `dotnet build -f netstandard1.6` will run only one fw (same for `dotnet run`, `test`, `publish`)

@polytypic i added some comment about this work in https://github.com/Hopac/Hopac/pull/123#issue-174564685